vic/drivers/shared_image/src/check_domain_info.c

```diff
@@ -58,7 +58,7 @@ compare_ncdomain_with_global_domain(char *ncfile)
     // loop over all grid cells and check that the two domains are identical
     for (i = 0; i < global_domain.ncells_total; i++) {
         // mask matches
-        if (ncfile_domain.locations[i].run > global_domain.locations[i].run) {
+        if (ncfile_domain.locations[i].run < global_domain.locations[i].run) {
```

Comment:
Oh I see - I was not sure what was being done here either. It would be nice to have a run flag that is separate from the mask, but maybe that is not necessary.